bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/Rule.java

```diff
@@ -345,7 +353,7 @@ public Module getModule(String moduleId) {
         } else if (Action.class == moduleClazz) {
             result = (List<T>) actions;
         }
-        return result != null ? result : new ArrayList<T>(11);
+        return result != null ? result : Collections.<T> emptyList();
```

Comment:
As I mention in daily, the list returned by the rule are modifiable for convenience to update the rule. For example if you want to add a new trigger to a rule, the new created trigger has to be added to the returned list instead of creating copy of existing ones and add the new rule. This can be done because the rule that is modified is your own copy and the changes will be available after call of ruleRegisrey.update(rule) method.